elyra/pipeline/processor_kfp.py

```diff
@@ -37,6 +38,30 @@ class KfpPipelineProcessor(PipelineProcessor):
     def type(self):
         return self._type
 
+    def get_auth_session_cookie(self, url, api_endpoint_username, api_endpoint_password):
+        get_response = requests.get(url)
```

Comment:
do we need this? or can we actually just assume you need to auth and avoid one extra hop to go directly to line 46?

Reply:
Won't this distinguish between the endpoints that are not secured, yet users have added a user/pwd to the config and those that are?  I think its probably a good idea to ensure 'auth' is present.

Reply:
I agree with Kevin, I made this as some kind of validation and not only rely on user config. If the first response stops at a redirected url where auth is required, we go ahead and authenticate